xbmc/windowing/gbm/WinSystemGbmGLESContext.h

```diff
@@ -39,6 +39,7 @@ class CWinSystemGbmGLESContext : public CWinSystemGbmEGLContext, public CRenderS
   void PresentRender(bool rendered, bool videoLayer) override;
 protected:
   void SetVSyncImpl(bool enable) override {}
+  std::unique_ptr<CVideoSync> GetVideoSync(void* clock) override;
```

Comment:
Sorry for a second round request but can you move this method to `CWinSystemGbm`. That way this will work for both GL and GLES rendering. Here is a patch to do so, https://termbin.com/5twt